carbon-design-system/carbon
# feat(react): deprecate `renderExpando` and add `renderToggle`

Closes #10317 

#### Changed
- Deprecated `renderExpando` in favor of `renderToggle` in AccordionItem so it wouldn't create a breaking change.
- Renamed internal `Expando` component to `Toggle` in AccordionItem for consistency.
- Replaced `enableExpando` with `enableToggle` in the DataTableExpansion story to finalize the deprecation started in the TableExpandHeader component.

#### Testing
- Make sure there are no breaking changes.
- Ensure Accordion and DataTableExpansion stories still work as expected.
